chain-impl-mockchain/src/testing/ledger.rs

```diff
@@ -100,6 +109,26 @@ impl ConfigBuilder {
             block0_date: Block0Date(0),
             consensus_version: ConsensusVersion::Bft,
             transaction_max_expiry_epochs: None,
+            #[cfg(feature = "evm")]
+            evm_params: EvmConfigParams {
+                config: EvmConfig::Istanbul,
+                environment: Environment {
+                    // FIXME: need to set a real price
+                    gas_price: GasPrice::default(),
+                    // Define the origin address with a random H160 address
+                    origin: Origin::random(),
+                    chain_id: ChainId::zero(),
+                    block_hashes: BlockHashes::new(),
+                    block_number: BlockNumber::zero(),
+                    block_coinbase: BlockCoinBase::zero(),
+                    block_timestamp: BlockTimestamp::zero() + 1,
+                    block_difficulty: BlockDifficulty::from(131_072),
+                    // FIXME: need to set a real limit
+                    block_gas_limit: GasLimit::max_value(),
+                    // FIXME: adequate base fee
+                    block_base_fee_per_gas: 1u32.into(),
+                },
```

Comment:
Hm, this one is a bit confusing. We are going to use this entire structure for blockchain configuration if I understood correctly. But some of these parameters can be derived from Jormungandr. Or maybe even should to avoid a strange/meaningless initial state. This effectively means we need two separate structures: one to configure the blockchain and the other to represent the environment state.

Reply:
Yes, the values from this were left as default values pending further definition, so it makes perfect sense to include it in future work.